pages/api/projects/[id]/index.js

```diff
@@ -16,4 +16,17 @@ export default async function handler(request, response) {
       return response.status(404).json({ status: "Page not found" });
     }
   }
+
+  if (request.method === "DELETE") {
+    try {
+      await Project.findByIdAndDelete(id);
+    } catch (error) {
+      console.error(`Can't delete project ${id}: ${error}`);
+    }
+
+    return response
+      .status(200)
+      .json({ status: "Project deleted successfully" });
+  }
```

Comment:
Handle it like:

```js
if (request.method === 'DELETE') {
  try {
    await Project.findByIdAndDelete(id);
    return response
      .status(200)
      .json({ status: 'Project deleted successfully' });

  } catch (error) {
    console.error(`Can't delete project ${id}`, error);
    return response
      .status(505)
      .json({ status: `Can't delete project ${id}` });
  } 
}
```

So you send the 500 in case you don't delete the project ^^